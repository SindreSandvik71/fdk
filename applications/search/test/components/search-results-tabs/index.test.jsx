import React from 'react';
import { shallow } from 'enzyme';
import ResultTabs from '../../../src/components/search-results-tabs';

test('should render ResultTabs with "Dataset" active correctly', () => {
  const defaultProps = {
    location: {
      pathname: '/',
      search: '',
      hash: '',
      key: 'ublg0e'
    }
  };
  const wrapper = shallow(<ResultTabs {...defaultProps} />);
  expect(wrapper).toMatchSnapshot();
});

test('should render ResultTabs with "Concepts" active correctly', () => {
  const defaultProps = {
    location: {
      pathname: '/concepts',
      search: '',
      hash: '',
      key: 'ublg0e'
    }
  };
  const wrapper = shallow(<ResultTabs {...defaultProps} />);
  expect(wrapper).toMatchSnapshot();
});
