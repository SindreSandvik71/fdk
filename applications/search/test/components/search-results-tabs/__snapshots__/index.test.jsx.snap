// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ResultTabs with "Concepts" active correctly 1`] = `
<div
  className="row"
>
  <div
    className="col-sm-12 col-md-8 col-md-offset-4"
  >
    <ul
      className="search-results-tabs"
    >
      <li
        className=""
      >
        <Link
          aria-label="Link til side for datasett:"
          replace={false}
          to={
            Object {
              "pathname": "/",
              "search": "?",
            }
          }
        >
          Datasett
          <span>
             (
            )
          </span>
        </Link>
      </li>
      <li
        className="li-active"
      >
        <Link
          aria-label="Link til side for begrep:"
          replace={false}
          to={
            Object {
              "pathname": "/concepts",
              "search": "?",
            }
          }
        >
          Begrep
          <span>
             (
            )
          </span>
        </Link>
      </li>
    </ul>
  </div>
</div>
`;

exports[`should render ResultTabs with "Dataset" active correctly 1`] = `
<div
  className="row"
>
  <div
    className="col-sm-12 col-md-8 col-md-offset-4"
  >
    <ul
      className="search-results-tabs"
    >
      <li
        className="li-active"
      >
        <Link
          aria-label="Link til side for datasett:"
          replace={false}
          to={
            Object {
              "pathname": "/",
              "search": "?",
            }
          }
        >
          Datasett
          <span>
             (
            )
          </span>
        </Link>
      </li>
      <li
        className=""
      >
        <Link
          aria-label="Link til side for begrep:"
          replace={false}
          to={
            Object {
              "pathname": "/concepts",
              "search": "?",
            }
          }
        >
          Begrep
          <span>
             (
            )
          </span>
        </Link>
      </li>
    </ul>
  </div>
</div>
`;
